docs/source/installation/2-manual-install.rst

```diff
@@ -111,6 +111,12 @@ reference.
 
         PasswordAuthentication yes
 
+    To allow keyboard interactive authentication, ensure
```

Comment:
I guess this is needed in specific cases and not all interactive authentication. I definitely could do an install without this change.

Found this useful reference: https://serverfault.com/questions/189643/sshd-blocking-password-authentication

Reply:
Yeah agreed it isn't necessary all of the time. Just checking that you agree that it isn't worth specifying that in the instructions given that it will be reverted in a later setup step anyway?

Reply:
I think if someone gets stuck at this step like you did, they are doing something special.

Seems fine to leave it if you think it could be useful for someone to make the initial root password step work, with a comment that "If needed you can enable keyboard interactive authentication with", though there are so many steps here that one would just go about copying pasting commands without really reading the instructions well, which is why it would be nice to keep commands limited and exclude anything special.
So, This could go either also go [here](https://commcare-cloud.readthedocs.io/en/latest/installation/troubleshooting.html)

Agreed that this later gets cleaned up.